sites/all/themes/orwell/sass/component/_audio-preview.scss

```diff
@@ -58,14 +58,14 @@
   font-family: $font-semibold;
   // To contain the button.
   // position: relative;
-  font-size: .8rem;
+  font-size: 0.8rem;
   display: flex;
   flex-flow: row wrap;
   justify-content: space-between;
   align-items: center;
 
   .audio-preview__time {
-    color: $color__gray;
+    color: $color__geist--grey5;
     flex: auto;
```

Comment:
<img width="531" alt="image" src="https://user-images.githubusercontent.com/15377965/81268005-a398cf80-9047-11ea-9c51-fdb1100b556c.png">

er det no-go at bruge color-geist--grey5 direkte? synes der er en tendens til at give dem nye navne inden man bruger dem...

Reply:
I do not have a problem with it.